profiles/sachinparihar.json

```diff
@@ -0,0 +1,7 @@
+{
+  "image": "https://user-images.githubusercontent.com/94243074/183848297-0cd1bd02-e62d-4ce0-90ec-3d557a394111.png",
+  "issueId": 1292,
+  "name": "Sachin Parihar",
+  "username": "sachinparihar"
+  }
+  
```

Comment:
There's an extra space here. GitHub doesn't display the endline.
```suggestion
}
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient